internal/server/backgroundjobs_test.go

```diff
@@ -29,15 +29,16 @@ func TestJobWrapper(t *testing.T) {
 	}
 
 	var runID int
-	job := func(ctx context.Context, tx *data.Transaction) error {
+	job := func(tx data.WriteTxn) error {
 		chReady <- signal
 		<-chRun
 
 		defer func() {
 			runID++
 		}()
 
-		tx = tx.WithOrgID(db.DefaultOrg.ID)
+		// nolint:forcetypeassert
+		tx = tx.(*data.Transaction).WithOrgID(db.DefaultOrg.ID)
```

Comment:
This was a consequence of changing the signature of the `jobs` function. Our production functions use `data.WriteTxn`, but our tests have to use `data.Transaction` so they can set the organization.  

It might be nice if `CreateIdentity` used the orgID on the passed in `Identity` instead of from the transaction. That would fix the problem without the type assert.

Reply:
👍  that would make things clearer

Reply:
I forgot this already does work, so I removed the forcetypeassert and changed to setting the orgID on the identity.